omod/src/main/java/org/openmrs/web/controller/user/UserFormController.java

```diff
@@ -284,6 +290,13 @@ public String handleSubmission(WebRequest request, HttpSession httpSession, Mode
 				us.changeQuestionAnswer(user, secretQuestion, secretAnswer);
 			}
 			
+			//Check if admin wants the person associated with the user to be added to the Provider Table
+			if(addToProviderTableOption != null) {
+				Provider provider = new Provider();
+				provider.setPerson(user.getPerson());
+				provider.setIdentifier(user.getSystemId());
```

Comment:
@wluyima Yeah. The system id is generated by the system automatically and acts as a good Identifier. The person who reported the issue from AMPATH specifically requested that we add the system id as identifier. You can find id in the description